rules/visual-design-specifications/rule.md

```diff
@@ -18,88 +18,93 @@ redirects:
 
-::: greybox
-**Incorporate branding:** It's important to incorporate the brand's visual identity into the design of a product! This should include the brand's given color scheme, typography, and logo at a minimum.
-:::
+**2. Determine the fidelity level** 
 
-::: good
-![Figure: Good example – High-fidelity mockup for ssw internship page](high-fidelity-mockup.jpg)
-:::
+You can use paper, a whiteboard, or digital tools depending on your preference. Understand the aim is to establish a shared understanding, rather than a polished artifact. Storyboard is meant to tell a story. Don’t spend too much time polishing the visuals unless you have to.
 
-### Interactive Fidelity
+Also See: [Do you determine specifications with a visual design(mockup rule with 3 level fidelity)?](https://www.ssw.com.au/rules/{New-Mockup-rules}/)
 
-To make a mockup more realistic and accurate to the end product, add interactivity to it. This includes consideration of how different elements will respond to user input, such as on-hover effects or the styling of visited links.
+**3. Define the user persona and scenario**
 
-`youtube: https://www.youtube.com/embed/-d6zNGeF59M`
+Understand the context in which your storyboard will be set. Identify the specific situation or scenario that your user will encounter, and define the characteristics and needs of the user persona involved.
 
-### Tools
+**4. Create visuals and add captions**
 
-* [Figma](https://www.figma.com) (Recommended)
-* [Sketch](https://www.sketchapp.com) (Mac Only and for UX designers)
-* [Moqups](https://moqups.com) (HTML5 based App)
-* [UXPin](http://uxpin.com) (more sophisticated, helps you create responsive designs)
+Begin sketching the visuals for each step of the storyboard. Use sketches to quickly depict a sequence or scene during brainstorming. Focus on capturing the essential elements that represent the actions, environment, and emotions of the user.
 
-### Summary
+**5. Present storyboard**
 
-Mockups help create more clarity than bulky design documents.
+Share your storyboard with the intended audience, which can be your internal team or project stakeholders, to effectively communicate your ideas and concepts.
 
-For example, to communicate that “a customer has many phone numbers”, a storyboard/wireframe of how that relationship will appear in the user interface is much more likely to be understood by the client.
+### What are the best tools?
 
-Clear communication of the message is more important than the medium used to convey that message.
+* [Figma](https://www.figma.com) (Recommended)
+* [Sketch](https://www.sketchapp.com) (Mac Only and for UX designers)
+* [Moqups](https://moqups.com) (HTML5 based App)
+* [UXPin](http://uxpin.com) (more sophisticated, helps you create responsive designs)
 
-::: greybox
+::: good
+![Figure: Good example – Example of a storyboard created in Figma](storyboarding-figma.png)
```

Comment:
I don't think this is a storyboard example, but a prototype...

> **Storyboarding** can strengthen the user experience elements of your designs. **Wireframe** is a low fidelity layout of the design. **Prototype** captures the intent of a design and simulates multiple states of that design.

Reply:
but I think this type of storyboard are most likely to use in SSW, 

a board of screen mockup linked with arrow to show how the software is going to be used.

Reply:
The concepts and terms can't be adapted to the way we may do stuff at SSW. These are public rules in the end.

Maybe just say "This prototype made in Figma is generated from a storyboard"

Reply:
changed to:

A prototype created in Figma can also be used effectively as a storyboard